From 489946d8f0e779d8ea79eac91a1aae350dec977b Mon Sep 17 00:00:00 2001
From: Artur Opalinski <ArturX.Opalinski@intel.com>
Date: Mon, 3 Aug 2020 11:37:20 +0000
Subject: [PATCH 07/17] Enable overlay filesystem for docker

Set CONFIG_OVERLAY_FS=m
Select other OVERLAY_FS params according to your needs, in our case it is:
	CONFIG_OVERLAY_FS=m
	CONFIG_OVERLAY_FS_REDIRECT_DIR=y
	CONFIG_OVERLAY_FS_REDIRECT_ALWAYS_FOLLOW=y
	+# CONFIG_OVERLAY_FS_INDEX is not set
	+# CONFIG_OVERLAY_FS_XINO_AUTO is not set
	CONFIG_OVERLAY_FS_METACOPY=y
---
 .config | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/.config b/.config
index 6f25b1ec2081..3847352c34b5 100644
--- a/.config
+++ b/.config
@@ -3968,7 +3968,12 @@ CONFIG_QUOTACTL_COMPAT=y
 CONFIG_AUTOFS4_FS=y
 CONFIG_AUTOFS_FS=y
 # CONFIG_FUSE_FS is not set
-# CONFIG_OVERLAY_FS is not set
+CONFIG_OVERLAY_FS=m
+# CONFIG_OVERLAY_FS_REDIRECT_DIR is not set
+CONFIG_OVERLAY_FS_REDIRECT_ALWAYS_FOLLOW=y
+# CONFIG_OVERLAY_FS_INDEX is not set
+# CONFIG_OVERLAY_FS_XINO_AUTO is not set
+# CONFIG_OVERLAY_FS_METACOPY is not set
 # CONFIG_SHIFT_FS is not set
 
 #
-- 
2.17.1

